.github/actions/restore/action.yml

```diff
@@ -49,26 +50,22 @@ runs:
     - name: Set app name variable
       shell: bash
       run: |
-        modified_environment=$(echo "${{ inputs.environment }}" | sed 's/_aks//g')
-        echo "modified_environment: ${modified_environment}"
-
-        if [ "${{ inputs.environment }}" == "review_aks" ] && [ "${{ inputs.pr-number }}" != "" ]; then
+        if [ "${{ env.app_environment }}" == "review" ] && [ "${{ inputs.pr-number }}" != "" ]; then
           echo "Set the app name to include the review app PR number."
-          app_name_suffix=${modified_environment}-${{ inputs.pr-number }}
-          echo "app_name=publish-${app_name_suffix}" >> $GITHUB_ENV
+          echo "app_name=publish-review-${{ inputs.pr-number }}" >> $GITHUB_ENV
         else
           echo "Set the app name to the environent name"
-          echo "app_name=publish-${modified_environment}" >> $GITHUB_ENV
+          echo "app_name=publish-${{ env.app_environment }}" >> $GITHUB_ENV
         fi
 
     - name: Install kubectl
-      if: ${{ inputs.environment != 'review_aks' }}
+      if: ${{ env.app_environment != 'review' }}
```

Comment:
what happens to the workflow if it's a review app?

Reply:
As per discussion